.github/workflows/production_deployment.yml

```diff
@@ -0,0 +1,100 @@
+name: Production Deployment
+
+on:
+  pull_request:
+    branches:
+      - main
+jobs:
+  accept:
+    # We want to be sure that an outside collaborator can't deploy using a fork, without us marking
```

Comment:
Can a label be applied on this repo by someone not on the `wps` team?

Reply:
Nope. People outside the team don't have rights to apply labels.